### PR TITLE
Fix autocast bug when schema is missing

### PR DESCRIPTION
**Pull request by gramalingam**
Autocast wasn't handling the case where a schema is missing correctly. Fix it.

### CODE REVIEW

File: onnxscript/autocast.py [hunk 1]
```diff
@@ -83,13 +83,11 @@ def cast(x, typeinfo) -> tensor.Tensor:
...
 
-    return cast_inputs(get_type_info, cast, op_schema, *args)
+    return cast_inputs(get_type_info, cast, op_schema, args)
 
 
-def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args):
+def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args) -> tuple[str, ...]:
     """Used for autocast during script-translation."""
-    if op_schema is None:
-        return args
```
Comment (justinchuby) [inline onnxscript/autocast.py:92]: Do we need a comment to remind future readers?

**Review by justinchuby — APPROVED** [review]
Thanks! 🚀

File: onnxscript/autocast.py [hunk 2]
```diff
@@ -83,13 +83,11 @@ def cast(x, typeinfo) -> tensor.Tensor:
             return tensor.Tensor(np.array(x, dtype=dtype))
         return x
 
-    return cast_inputs(get_type_info, cast, op_schema, *args)
+    return cast_inputs(get_type_info, cast, op_schema, args)
 
 
-def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args):
+def static_cast_inputs(converter, op_schema: Optional[OpSchema], *args) -> tuple[str, ...]:
```
Comment (justinchuby) [inline onnxscript/autocast.py:89]: Should this take `args` instead of `*args` too?

Comment (gramalingam) [reply onnxscript/autocast.py]: Done

**Pull request merged**